src/graphql/resolvers/classResolver.js

```diff
@@ -44,7 +49,7 @@ const Class = {
     ];
 
     if (args.name) {
-      const filter = resolveNameFilter(args.name);
+      const filter = resolveContainsStringFilter(args.name);
       filters.push(filter);
```

Comment:
One more cleanup. Is there a reason we can't just do this?
```js
filters.push(resolveNameFilter(args.name)):
```

Reply:
I could do that.